WEB/Src/DependencyCollector/DependencyCollector/Implementation/AzureSdk/AzureSdkDiagnosticsEventHandler.cs

```diff
@@ -15,6 +15,7 @@
 
     internal class AzureSdkDiagnosticsEventHandler : DiagnosticsEventHandlerBase
     {
+        private const string CosmosDBResourceProviderNs = "Microsoft.DocumentDB";
```

Comment:
Is DocumentDB instead of CosmosDB intentionally done for UI to show icons? If yes, please add a small comment above.

Reply:
Yes, this is intentional, comment added